app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ProfileController < ApplicationController
+  before_action :authenticate_user!
+
+  def edit
+    render :edit, locals: { user: current_user }
+  end
+
+  def update
+    user = current_user
```

Comment:
May I ask what's the point of this assignment here? From my standpoint of view that's not necessary